tests/pathfinding/test_graphs.py

```diff
@@ -185,3 +186,75 @@ def get_paths(diversity_penalty):
         [0, 7, 9, 10, 8],
         [0, 7, 6, 5, 8],
     ]
+
+
+@pytest.mark.usefixtures("populate_token_network_case_3")
+def test_reachability_initiator(token_network_model: TokenNetwork, addresses: List[Address]):
+    def get_paths():
+        paths = token_network_model.get_paths(
+            addresses[0], addresses[8], value=TokenAmount(10), max_paths=5
+        )
+        index_paths = [addresses_to_indexes(p["path"], addresses) for p in paths]
+        return index_paths
+
+    assert get_paths() == [
+        [0, 7, 8],
+        [0, 1, 2, 3, 4, 8],
+        [0, 7, 6, 8],
+        [0, 7, 9, 10, 8],
+        [0, 7, 6, 5, 8],
+    ]
+
+    token_network_model.address_to_reachability[addresses[0]] = AddressReachability.UNREACHABLE
+    assert get_paths() == []
+
+    token_network_model.address_to_reachability[addresses[0]] = AddressReachability.UNKNOWN
+    assert get_paths() == []
+
+
+@pytest.mark.usefixtures("populate_token_network_case_3")
+def test_reachability_mediator(token_network_model: TokenNetwork, addresses: List[Address]):
+    def get_paths():
+        paths = token_network_model.get_paths(
+            addresses[0], addresses[8], value=TokenAmount(10), max_paths=5
+        )
+        index_paths = [addresses_to_indexes(p["path"], addresses) for p in paths]
+        return index_paths
```

Comment:
`get_paths` is repeated a few times. We should pull that out into a separate fixture.

Reply:
Ref https://github.com/raiden-network/raiden-services/issues/349